boards/DCM/Src/main.c

```diff
@@ -187,11 +190,15 @@ int main(void)
 
     clock = App_SharedClock_Create();
 
+    imu = App_Imu_Create(
```

Comment:
nit; similar to previous comments. Move below buzzer to group this with objects

Reply:
Ok gotcha